contracts/connection-router/src/types.rs

```diff
@@ -83,39 +83,14 @@ impl KeyDeserialize for MessageID {
 
 /// cosmwasm cannot serialize tuples, so we need to convert [GlobalMessageId] into a struct
 #[cw_serde]
-struct GlobalMessageIdSerde {
-    chain_name: ChainName,
-    message_id: MessageID,
-}
-
-#[cw_serde]
-#[serde(from = "GlobalMessageIdSerde", into = "GlobalMessageIdSerde")]
-pub struct GlobalMessageId(ChainName, MessageID);
-
-impl From<GlobalMessageId> for GlobalMessageIdSerde {
-    fn from(other: GlobalMessageId) -> Self {
-        Self {
-            chain_name: other.0,
-            message_id: other.1,
-        }
-    }
-}
-
-impl From<GlobalMessageIdSerde> for GlobalMessageId {
-    fn from(other: GlobalMessageIdSerde) -> Self {
-        Self(other.chain_name, other.message_id)
-    }
-}
-
-impl GlobalMessageId {
-    pub fn new(chain_name: ChainName, message_id: MessageID) -> Self {
-        Self(chain_name, message_id)
-    }
+pub struct GlobalMessageId {
+    pub chain: ChainName,
+    pub message_id: MessageID,
```

Comment:
```suggestion
pub struct CrosschainUniqueId {
    pub chain: ChainName,
    pub message_id: ChainUniqueId,
```
I prefer calling things what they are instead of what they are for.